src/ui/server.js

```diff
@@ -22,25 +22,30 @@ export default function render(
 
   function renderStandardRoute() {
     const bundlePath = locals.assets["main"];
-    const {html, css} = StyleSheetServer.renderStatic(() =>
-      ReactDOMServer.renderToString(
+    // Reference for MUI stylesheet injections into SSR HTML:
+    // https://material-ui.com/guides/server-rendering/#handling-the-request
+    const sheets = new ServerStyleSheets();
+    const html = ReactDOMServer.renderToString(
+      sheets.collect(
         <StaticRouter location={path} context={context}>
           <App />
         </StaticRouter>
       )
     );
+
+    // This _must_ follow sheets.collect, or `css` will be empty
+    const css = sheets.toString();
+
     const page = dedent`\
       <!DOCTYPE html>
       <html>
       <head>
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width,initial-scale=1" />
       <link rel="shortcut icon" href="${assets.resolve("/favicon.png")}" />
-      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-      <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
       <title>SourceCred</title>
       <style>${require("./index.css")}</style>
-      <style data-aphrodite>${css.content}</style>
+      <style id="jss-server-side">${css}</style>
```

Comment:
We still use Aphrodite, so let’s keep both? I expect that it should be
possible to nest the callbacks.

Reply:
Yup! having the nested callbacks works fine